README.md

```diff
@@ -21,247 +18,160 @@ techniques for pre-rendering cacheable pages.
+>
+> - To choose a specific version, to to: https://adoptium.net/temurin/releases/
+> - If you're on a Mac with an Apple Silicon M1 or M2 chip, choose the `aarch64` architecture.
+> - Choose `JRE` instead of the full `JDK` package unless you plan to do Java
+>   development.
 
-## Tests
+### Run the app locally
 
-```bash
-npm test
-npm run test:watch
-```
+1. Start the Firestore Emulator
 
-[ts-jest](https://kulshekhar.github.io/ts-jest/) is installed as a dev
-dependency to support writing tests in TypeScript. The jest configuration
-(`jest.config.js`) sets `preset: 'ts-jest'` to enable this.
+   Choose the port you want to use for the [Firestore Emulator], for example `9999`.
 
-Jest will run tests located in `__test__` directories (for example:
-`src/lib/__test__/database.test.ts`).
+   > Note: If you choose a port that is not `9999` for this demo, you will need to update the `host` property in `database.ts` to match your selected port.
 
-## Deployment
+   Export the `FIRESTORE_EMULATOR_HOST` environment variable:
 
-Ready to deploy? Check out how to do that [here](/terraform/README.md). 
+   ```bash
+   export FIRESTORE_EMULATOR_HOST="localhost:9999"
+   ```
 
-## Prerequisites
+   When the exported `FIRESTORE_EMULATOR_HOST` environment variable is set, the
+   Firestore client will automatically use it to connect to the emulator.
 
-You need a Google Cloud account if you want to open the demo in [Cloud Shell]
-(a free online development and operations environment with an integrated editor)
-and deploy it.
+   To start the emulator:
 
-If you want to explore the developer experience **using your own machine**, you
-will need to install the Google Cloud CLI.
+   ```bash
+   gcloud emulators firestore start --host-port="$FIRESTORE_EMULATOR_HOST" --project=demo-test
+   ```
 
-While testing the app you can connect to a hosted Cloud Firestore database or
-you can run the [Firebase Emulator].
+   Press `Ctrl-C` when you want to stop the Firestore emulator.
 
-* [Node.js] (installing the `LTS` version is recommended).
-* [Install the Google Cloud CLI](#3-install-the-google-cloud-cli).
-* [Install the Firestore Emulator](#4-install-the-firestore-emulator) (optional).
+2. Get the code
 
+   Clone this repo and change your current working directory to it:
 
-### 1. Sign up for a Google Cloud Account
+   ```bash
+    git clone https://github.com/googlecloudplatform/developer-journey-app.git
+    cd developer-journey-app
+   ```
 
-You can [sign up here](https://accounts.google.com/SignUp).
+3. Set local development environment variables
 
-Once you have an account, you'll have access to the following:
+   Copy the example `.env.local.example` file to a new file `.env.local`:
 
-- [Free Trial]
-- [Free Tier]
+   ```bash
+   cp .env.local.example .env.local
+   ```
 
-The Free Trial provides $300 of Cloud Billing credits to pay for resources for
-90 days while you try out Google Cloud.
+   If neccessary, change any of the default values provided.
```

Comment:
I'd mention something additional like: 
"This file is picked up by PACKAGE automatically, and sets a number of defaults, including FIREBASE_EMULATOR_HOST." (Where PACKAGE is whatever part of the stack uses the local env file.). This will help clarify why this envvar needs to be re-declared elsewhere.